service-worker/README.md

```
@@ -0,0 +1,33 @@
+
+```
+npm i @hoprnet/uhttp-lib
+```
+
+The package has to be added to main dependencies and not the devDependencies.
+
+Add `uhttp-post-build-react" to your build process right after the build is finished:
+
+eg:
+
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)